web_src/js/features/admin/common.ts

```diff
@@ -24,7 +25,7 @@ export function initAdminCommon(): void {
   initAdminAuthentication();
   initAdminNotice();
 
-  queryElems(document, '.avatar-file-with-cropper', initAvatarUploaderWithCropper);
+  queryElems(el, '.avatar-file-with-cropper', initAvatarUploaderWithCropper);
```

Comment:
It could use the global-init framework, then these calls to `queryElems` could be removed.